include/up-cpp/datamodel/validator/UMessage.h

```diff
@@ -46,7 +46,9 @@ enum class Reason {
 	/// @brief The Request ID did not match the ID of the request message
 	REQID_MISMATCH,
 	/// @brief The Priority did not match the Priority of the request message
-	PRIORITY_MISMATCH
+	PRIORITY_MISMATCH,
+	/// @brief The source and sink from a request must be swapped in a response
+	URI_MISMATCH
```

Comment:
You added this error condition, but didn't use it.  Was that an oversight or intentional ?

Reply:
It _should_ be used :thinking:

Reply:
[Good catch](https://github.com/eclipse-uprotocol/up-cpp/pull/166/files#r1651456460)

Reply:
A corresponding test should also be added :sweat_smile:

Reply:
Done